Gallery of Horns/src/Component/HornedBeast.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+class HornedBeast extends React.Component{
+  constructor(props){
+    super(props);
+  }
+
+  render(){  
+    return(
+      <div>
+        <h2>{this.props.title}</h2>
+        <img title={this.props.title} src={this.props.imageURL} alt={this.props.description}/> 
```

Comment:
Hey Bradley! Images can be tricky to render in React, but I did want to point out that the props you are using for the src has a small typo. In the Gallery.jsx file, that class is passing into each HornedBeast component a prop called `imageUrl` and you are calling `this.props.imageURL` when it should be `this.props.imageUrl`.

Reply:
Ah. Thank you! That must be why I was having trouble. Before, I even did the same technique you mentioned below. I will implement this. Thanks!